### PR TITLE
Rename hull to point in Ethereum boot logic

### PR DESCRIPTION
**Pull request by Fang-**
Matches the changes in urbit/arvo#946. Here it's just a simple renaming.

### CODE REVIEW

**Comment by joemfb** [comment]
This looks right to me. When urbit/arvo#946 settles down, I'll try it against ropsten or just a ganache node with the latest contracts.

File: vere/dawn.c [hunk 1]
```diff
@@ -299,21 +299,21 @@ u3_dawn_vent(u3_noun seed)
   }
 
   {
-    //  +hull:constitution:ethe: on-chain state
+    //  +point:constitution:ethe: on-chain state
     //
     u3_noun hul;
```
Comment (joemfb) [inline vere/dawn.c]: Please rename to `pon` or somesuch.

**Comment by Fang-** [comment]
Would you like to see the pinned arvo commit updated to match urbit/arvo#946, even though it doesn't impact CI? On the one hand, bookkeeping, on the other hand, "just use latest".

**Comment by joemfb** [comment]
There's some issues with stale userspace in the CI-built pills, and CI here uses those pill exclusively, without a local Arvo checkout. Let's skip the update for now, I'll work on those pills today.

File: vere/dawn.c [hunk 2]
```diff
@@ -368,12 +368,12 @@ u3_dawn_vent(u3_noun seed)
     }
 
     //  (unit ship): sponsor
-    //  produced by +veri:dawn to avoid coupling to +hull structure
+    //  produced by +veri:dawn to avoid coupling to +point structure
     //  XX reconsider
     //
     pon = u3k(u3t(sas));
```
[the remark this answers is not in the document]
Comment (Fang-) [reply vere/dawn.c]: Also my bad for not checking thoroughly.

_picks up like twenty balls off the floor_

**Pull request merged**